src/components/StickerTemplate.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {
+    View,
+    Text,
+    Image,
+    StyleSheet,
+    ImageBackground,
+    Dimensions,
+} from 'react-native';
+// Imagenes de Plantilla
+import fondo from '../../assets/appAssets/template/fondo.jpg';
+import marco from '../../assets/appAssets/template/marco.png';
+import delantero from '../../assets/appAssets/test/delantero.png';
+import defensa from '../../assets/appAssets/test/defensa.png';
+import medioCentro from '../../assets/appAssets/test/medioCentro.png';
+import arquero from '../../assets/appAssets/test/arquero.png';
+
+const { width } = Dimensions.get('window');
+
+// Mientras no se tenga banderas
+import bandera from '../../assets/appAssets/test/bandera.png';
+
+
+export default function StickerTemplate({ sticker }) {
+    return (
```

Comment:
Hook personalizado para el role Image
                            
```suggestion

function usePlayerRoleImg(sticker) {
   return () => {
          switch (sticker) {
              case 'Arquero': 
                  return arquero;
               break;
               .... // hacer lo mismo con el resto
          }
          
          return delantero; //default?
   }
}

export default function StickerTemplate({ sticker }) {
    const playerRoleImg = usePlayerRoleImg(sticker);

    return (
```